### PR TITLE
[ES] Update `Internet Relay Chat`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/Internet_Relay_Chat/es.md [hunk 1]
```diff
@@ -1,81 +1,86 @@
...
-- **Puerto:** `6667` (por defecto)
-- **Nombre de usuario:** Tu nombre de usuario en osu! (sustitute espacios por guiones bajos)
-- **Contraseña:** Consiguela en [Autenticación IRC](https://osu.ppy.sh/p/irc).
+- Servidor: `irc.ppy.sh`
+- Puerto: `6667`
+- Nombre de usuario: tu nombre de usuario de osu!. Reemplaza los espacios con guiones bajos (por ejemplo, `beppy master 1000` se convierte en `beppy_master_1000`)
+- Contraseña: la contraseña de la [página de configuración de la cuenta](https://osu.ppy.sh/home/account/edit#legacy-api)
 
-*Tu contraseña IRC es diferente de la contraseña de la cuenta **No la compartas con otros**.*
+*Advertencia: Tu contraseña de IRC es diferente a la contraseña de tu cuenta. **No lo compartas con otros**.*
```
Comment (UnknownOsu) [inline wiki/Community/Internet_Relay_Chat/es.md:22]:
```suggestion
*Advertencia: Tu contraseña de IRC es diferente a la contraseña de tu cuenta. **No la compartas con otros**.*
```

File: wiki/Community/Internet_Relay_Chat/es.md [hunk 2]
```diff
@@ -1,81 +1,86 @@
...
-```
+| Cliente de IRC | Descripción |
+| :-- | :-- |
+| [HexChat](https://hexchat.github.io/) | Marca «Hide join and part messages» en `Settings` -> `Preferences` -> `Chatting` -> `General` (Advanced pre-2.9.6)  |
+| [ircII](http://www.eterna.com.au/ircii/) | Escribe `IGNORE * CRAP` |
+| [Irssi](https://irssi.org) | Escribe `/ignore * JOINS PARTS QUITS` |
+| [Weechat](https://weechat.org/) | Escribe `/filter add joinquit * irc_join,irc_part,irc_quit *` |
+| [Konversation](https://konversation.kde.org/) | Marca «Hide Join/Part/Nick Events» en `Settings` -> `Configure Konversation...` (`Ctrl` + `Shift` + `,`) -> `Behavior` -> `Chat Window` |
+| [KVIrc](https://www.kvirc.net/) | (Consulta [la página de la wiki de KVIrc](https://github.com/kvirc/KVIrc/wiki/FAQ#how-do-i-suppress-join-part-and-quit-messages)) |
+| [mIRC](https://www.mirc.com/) | Ve a mIRC Options (`Tools` -> `Options` o `Alt` + `O`), selecciona `IRC`, haz clic en el botón `Events...` y selecciona `Hide` para uniones y partes |
```
Comment (UnknownOsu) [inline wiki/Community/Internet_Relay_Chat/es.md:48]:
```suggestion
| [mIRC](https://www.mirc.com/) | Ve a mIRC Options (`Tools` -> `Options` o `Alt` + `O`), selecciona `IRC`, haz clic en el botón `Events...` y selecciona `Hide` para uniones y partidas |
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Community/Internet_Relay_Chat/es.md (hunk 2 above)
Comment (agatemosu) [reply wiki/Community/Internet_Relay_Chat/es.md:48]: Es un parte https://datatracker.ietf.org/doc/html/rfc1459#section-4.2.2

Comment (UnknownOsu) [reply wiki/Community/Internet_Relay_Chat/es.md:48]: Creo que se refiere a un mensaje de partida, como si estuviera dejando el canal

**Review by TicClick — APPROVED** [review]

**Pull request merged**